whylog/config/__init__.py

```diff
@@ -125,8 +126,16 @@ def _load_rules(self):
         ]
 
     def _load_log_types(self):
-        #TODO fix log types loading
-        pass
+        matchers = defaultdict(list)
+        matcher_definitions = self._load_file_with_config(self._log_type_path)
+        matchers_factory_dict = {'RegexFilenameMatcher': RegexFilenameMatcherFactory}
+        for definition in matcher_definitions:
+            matcher = matchers_factory_dict[definition['matcher_class_name']].from_dao(definition)
```

Comment:
`KeyError: no such key 'WildcardFilenameMatcher' in matchers_factory_dict` will happen here one day and you should handle it.

In the future one user can have a higher whylog version than another one. The error displayed here should inform the user of the old version, that the configuration specified an unsupported filename matcher, so the user should consider upgrading his whylog version.